src/main/java/org/apache/sysds/runtime/matrix/operators/AggregateOperator.java

```diff
@@ -31,7 +29,7 @@
 import org.apache.sysds.runtime.functionobjects.ValueFunction;
 
 
-public class AggregateOperator  extends Operator implements Serializable
```

Comment:
this would need stay serializable - otherwise lazy evaluation can create funny exceptions in Spark.